kernelci/build.py

```diff
@@ -1098,10 +1098,14 @@ def _create_cros_config(self, config):
             raise FileNotFoundError("Error reading {}".format(url))
         tar = tarfile.open(cros_config)
         with open(os.path.join(self._output_path, ".config"), 'wb') as f:
-            f.write(tar.extractfile("base.config").read())
-            f.write(tar.extractfile(os.path.join(os.path.dirname(config),
-                                    "common.config")).read())
-            f.write(tar.extractfile(config).read())
+            config_file_names = [
+                'chromeos/base.config',
+                os.path.join(
+                    'chromeos', os.path.dirname(config), "common.config"),
+                os.path.join('chromeos', config),
+            ]
+            for file_name in config_file_names:
+                f.write(tar.extractfile(file_name).read())
```

Comment:
Maybe instead of having hardcoded `'chromeos'` string we can have it passed as a keyword parameter to the `_create_cros_config()` method.

e.g.

```python
def _create_cip_config(self, config, config_root='chromeos')
```

Reply:
Yes, we could also have the list of config files passed as a parameter so whenever the contents of config archive change we only have to update some config and not the code.

But at the moment, the config file names are hard-coded, so the fact they're now in a sub-directory is kind of a detail in the path used to find them. So I think these are different issues.

While we keep this code on the `chromeos` branch, I'm more in favour of having it as small and simple as possible to minimise potential rebase conflicts.

Reply:
I thought of having string  `'chromeos`' defined in one place rather than repeated in three, but the fact of minimizing potential rebase conflicts convinces me.

Reply:
OK I've added a `subdir = 'chromeos'` local variable to avoid repeating 'chromeos' while keeping the code changes inside the method only.